src/Soil/Soil.jl

```diff
@@ -86,8 +86,9 @@ export RichardsModel,
     AbstractSoilModel,
     AbstractSoilSource,
     FluxBC,
-    FreeDrainage,
```

Comment:
keep export?

Reply:
i think it appears twice in the list, see above